### PR TITLE
Newsletter 10: Add Jobs and ToC

### LINKED ISSUE #150: Jobs Section
We don't have a policy regarding job offer posts yet and there was some negative feedback about the [job ads in previous newsletter issues](https://rust-gamedev.github.io/posts/newsletter-009/#dungeonfog-is-hiring). 

_from the WG channel on Discord:_

> **aclysma**: I have mixed feelings about a hiring post being the first thing on the newsletter
> **ozkriff**: why? seems like important information to me: opportunities to work on graphics stuff using Rust/wgpu are quite rare atm
> **aclysma**: commercial entities have a natural advantage over non-commercial in marketing.. and the focus being on their company hiring someone instead of how their company is pushing rust game development forward seems more to serve their company's interests than the communities interest
> I'm certainly not saying a hiring post has no value to the community or that it shouldn't be included,  I'd just rather see the top spot go to something that's in service to the community rather than the other way around
> If it was a video of some cool new features with an oh-btw-we're-hiring blurb at the end (i.e. the focus being on showing something new and interesting) then my opinion would probably be different
> **ozkriff**: Will it be better if i swap the hiring section with london talks?
> **aclysma**: I think it would
> or pick a game update to move up
> **XAMPPRocky**: I also agree with moving the job offer down, reading it, it felt like a ad to me.

So I propose adding an optional first-level "Jobs" section (between "Requests for Contribution" and "Bonus"). Any thoughts/opinions/objections? :)

### PR DESCRIPTION
**Pull request by ozkriff**
Closes #150 (_"Jobs Section"_) 
Part of #139 (_"Newsletter 10: Coordination/Tracking"_)
Related to #38 (_"Add table of contents"_)

### CODE REVIEW

**Comment by ozkriff** [comment]
@hakanm r?

File: content/posts/newsletter-010/index.md [hunk 1]
```diff
@@ -541,6 +541,30 @@ or [join the next meeting][join].
 [abstreet-issues]: https://github.com/dabreegster/abstreet/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
 [mun-issues]: https://github.com/mun-lang/mun/labels/good%20first%20issue
 
+## Jobs
+
+<!-- An optinal section for new jobs related to Rust gamedev -->
+
+- [Elektron][elektron] (synth manufacturer) looking to hire Rust Graphics Engineer
+  (Gotheburg, Sweden):
```
Comment (hakanm) [inline content/posts/newsletter-010/index.md:561]: Gothenburg is misspelled

Comment (ozkriff) [reply content/posts/newsletter-010/index.md]: Copypasted from [TWIR](https://this-week-in-rust.org/blog/2020/05/27/this-week-in-rust-340/). Fixed, thanks!

File: newsletter-template.md [hunk 2]
```diff
@@ -71,6 +71,10 @@ or [join the next meeting][join].
 
 <!-- Links to "good first issue"-labels or direct links to specific tasks -->
 
+## Jobs
+
+<!-- An optinal section for new jobs related to Rust gamedev -->
```
Comment (17cupsofcoffee) [inline newsletter-template.md:76]:
```suggestion
<!-- An optional section for new jobs related to Rust gamedev -->
```

**Pull request merged**